tee-worker/litentry/primitives/src/lib.rs

```diff
@@ -36,5 +45,138 @@ pub use parentchain_primitives::{
 	Signature as ParentchainSignature, UserShieldingKeyNonceType, UserShieldingKeyType, VCMPError,
 	Web3Network, ASSERTION_FROM_DATE, MAX_TAG_LEN, MINUTES, NONCE_LEN, USER_SHIELDING_KEY_LEN,
 };
+use scale_info::TypeInfo;
+use sp_core::{crypto::AccountId32, ecdsa, ed25519, sr25519, ByteArray, H160};
+use sp_io::{crypto::secp256k1_ecdsa_recover, hashing::keccak_256};
+use sp_runtime::traits::{BlakeTwo256, Verify};
+
+pub type HashedAddressMapping = GenericHashedAddressMapping<BlakeTwo256>;
+
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum LitentryMultiAddress {
```

Comment:
Do you consider having a conversion between `LitentryMultiAddress` and `Identity`? Not strictly needed tho

Reply:
I replaced `LitentryMultiAddress` with `Identity`